config/config_helpers.go

```diff
@@ -157,12 +158,18 @@ func getParentTerragruntDir(include *IncludeConfig, terragruntOptions *options.T
 
 func parseGetEnvParameters(parameters []string) (EnvVar, error) {
 	envVariable := EnvVar{}
-	if len(parameters) != 2 {
-		return envVariable, errors.WithStackTrace(InvalidGetEnvParams{ExpectedNumParams: 2, ActualNumParams: 2, Example: `getEnv("<NAME>", "<DEFAULT>")`})
-	}
 
 	envVariable.Name = parameters[0]
-	envVariable.DefaultValue = parameters[1]
+
+	if len(parameters) == 1 {
+		envVariable.IsRequired = true
+	} else {
```

Comment:
Should move to switch case of `parameters` and `default` case is

```
return envVariable, errors.WithStackTrace(InvalidGetEnvParams{ExpectedNumParams: 2, ActualNumParams: 2, Example: `getEnv("<NAME>", "<DEFAULT>")`})
```

Reply:
+1 to use `switch` case here.

Reply:
Done, thanks for the hint